app/server/api-generator/src/test/java/io/syndesis/server/api/generator/soap/AbstractSoapExampleTest.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.server.api.generator.soap;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import io.syndesis.common.model.connection.Connector;
+import io.syndesis.common.model.connection.ConnectorSettings;
+import io.syndesis.common.util.IOStreams;
+import io.syndesis.common.util.json.JsonUtils;
+import io.syndesis.server.api.generator.ConnectorGenerator;
+import io.syndesis.server.api.generator.openapi.TestHelper;
+import io.syndesis.server.api.generator.soap.parser.SoapApiModelParserTest;
+
+import static io.syndesis.server.api.generator.soap.SoapConnectorConstants.*;
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Base class for example WSDL driven tests.
+ */
+@RunWith(Parameterized.class)
+public abstract class AbstractSoapExampleTest {
+
+    protected final String specification;
+    protected final ConnectorGenerator connectorGenerator;
+
+    public AbstractSoapExampleTest(String resource) throws IOException, InterruptedException {
+        if (resource.startsWith("http")) {
+            // add a slight delay to avoid DDoS behavior on server
+//            Thread.sleep(100);
```

Comment:
```suggestion
```

Reply:
Removed. I was getting timeouts from Workday when I was testing against all of it's WSDL endpoints, and had to add the delay to avoid that.